eclass/user.eclass

```diff
@@ -512,4 +512,69 @@ esethome() {
 	esac
 }
 
+# @FUNCTION: esetshell
+# @USAGE: <user> <shell>
+# @DESCRIPTION:
+# Update the shell in a platform-agnostic way.
+# Required parameters is the username and the new shell.
+# Specify -1 if you want to set shell to platform-specific nologin.
+esetshell() {
```

Comment:
Up to you, but I would default to making these private functions in the new `acct-user.eclass`. While they do fit perfectly in `user.eclass` today, they should only ever be used by `acct-user.eclass`, and I'm assuming that some day we'll want to deprecate `user.eclass`. When that day comes, we'll have one less function to move, and we will have ensured that no calls to `esetshell` have appeared in random ebuilds in the meantime.

Reply:
The way I see it, we're going to block using user.eclass outside acct-* categories when we're done. I don't think we really need to be stronger about it today.